client/apps/settings/client/curations/venice_section.coffee

```diff
@@ -0,0 +1,114 @@
+React = require 'react'
+ReactDOM = require 'react-dom'
+moment = require 'moment'
+{ div, input, label, span, textarea } = React.DOM
+AutocompleteList = React.createFactory require '../../../../components/autocomplete_list/index.coffee'
+imageUpload = React.createFactory require '../../../edit/components/admin/components/image_upload.coffee'
+
+module.exports = VeniceSection = React.createClass
+  displayName: 'VeniceSection'
+
+  getInitialState: ->
+    title: @props.section.title or ''
+    description: @props.section.description or ''
+    video_url: @props.section.video_url or ''
+    artist_ids: @props.section.artist_ids or []
+    cover_image: @props.section.cover_image or ''
+    published: @props.section.published or false
+    release_date: moment(@props.section.release_date).format('YYYY-MM-DD') or null
+
+  componentDidMount: ->
+    ReactDOM.findDOMNode(@refs.container).classList += ' active'
+
+  onInputChange: (e) ->
+    @onChange e.target.name, e.target.value
+
+  onChange: (key, value) ->
+    newState = @state
+    newState[key] = value
+    @setState newState
+    @props.onChange newState, @props.id
+
+  onCheckboxChange: (e) ->
+    @onChange 'published', !@state.published
+
+  onDateChange: (e) ->
+    @onChange 'release_date', moment(e.target.value).toISOString()
+
+  render: ->
+    div { className: 'venice-admin__fields', ref: 'container'},
+      div { className: 'fields--full'},
+        div { className: 'fields--left' },
+          div { className: 'field-group' },
+            label {},'Title'
+            input {
+              className: 'bordered-input'
+              placeholder: 'Title'
+              defaultValue: @state.title
+              onChange: @onInputChange
+              name: 'title'
+            }
+          div { className: 'field-group' },
+            label {},'Video URL'
+            input {
+              className: 'bordered-input'
+              placeholder: 'Enter a link'
+              defaultValue: @state.video_url
+              onChange: @onInputChange
+              name: 'video_url'
+            }
```

Comment:
I'm sure there will be more updates to other sections, but I'll just note anyway that 
 there's a high chance this field will need to accept multiple files so we can optimize the video for device/bandwidth. Def not a blocker atm tho